### ISSUE TITLE
default aom version doesn't exist

### ISSUE THREAD

**Issue by cameroncan** [SoftCreatR/imei#10]
## 🐛 Bug Report

When using the default configuration the imei.sh script fails because the aom tar file doesn't exist. The version returned by https://raw.githubusercontent.com/SoftCreatR/imei/main/versions/aom.version (2.1.01) isn't an available version at https://github.com/jbeich/aom/releases. Providing for the argument --aom-version 2.1.0-rc1 worked great. 

## Have you spent some time to check if this issue has been raised before?

Yes

## To Reproduce

Use the recommended command ./imei.sh

## Expected behavior

Successful installation

## Actual Behavior

Failed on the "Building aom step"

## Your Environment

Using the latest git repo code, ubuntu 20.04

**Comment by REJack** [comment]
Try to add the `--aom-version` option like `bash <(wget -qO - dist.1-2.dev/imei) --aom-version 2.0.2`.

**Comment by chrispy-snps** [comment]
First, thanks VERY much for developing IMEI!

Ubuntu 20.04 here, and I think I'm hitting the same issue. From the log:

```
Connecting to codeload.github.com (codeload.github.com)|140.82.112.9|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-03-14 22:20:58 ERROR 404: Not Found.

./imei.sh: line 342: [: da39a3ee5e6b4b0d3255bfef95601890afd80709  aom-2.1.01.tar.gz: integer expression expected
tar: This does not look like a tar archive

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

I tried adding `--aom-version 2.0.2` per your advice, and it worked! Although, I have no idea what an `aom` is or how I would find the latest version (or at least what version I'd want to specify).

**Comment by SoftCreatR** [comment]
Hey,

thank you for your bug report. There seems to be a bug in the version updater. And due to the fact, that aom's versioning has been changed, an update of the version updater is required either way.

**Comment by SoftCreatR** [comment]
The issue should be fixed via 7a4005e57f24ecb68b8fcc22571b75009139d50e by replacing the whole version update script/logic.

**Comment by chrispy-snps** [comment]
The latest version works perfectly on Ubuntu 20.04 now. Thanks!
